data/scripts/install-man-pages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/env sh
+
+# Set PREFIX.
+if [ -z  "${1+x}" ]; then
+    if ! [ -z "${PREFIX+x}" ]; then
+        PREFIX=/usr
+    fi
+else
+    PREFIX=${1}
+fi
+export PREFIX
+
+echo "Installing man pages"
+install -d ${PREFIX}/share/man/man1
+install -m 644 data/man/tilix ${PREFIX}/share/man/man1/tilix.1
```

Comment:
Change `-m` to `-Dm`, see the previous comment.
`${PREFIX}` should be quoted, see comment above.